### ISSUE TITLE
inconsistent depth size in DUT-RGBD-400*600

### ISSUE THREAD

**Issue by ShuhanChen** [jiwei0921/DMRA_RGBD-SOD#6]
I found the depth size is 256x256 in DUT-RGBD-400*600, can you provide the original depth file (400x600)?

**Comment by jiwei0921** [comment]
Of course, I am sorry for the late reply. This link (link: https://pan.baidu.com/s/1t1fko_soLa6VlCGoPgQREA  code: eixe) is available.
